code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -147,7 +147,7 @@
 	var/list/adjacent_turfs = atmos_adjacent_turfs
 	var/datum/excited_group/our_excited_group = excited_group
 	var/adjacent_turfs_length = LAZYLEN(adjacent_turfs)
-	atmos_cooldown++
+	var/cached_atmos_cooldown = atmos_cooldown + 1
```

Comment:
That would imply a write back into the var though. And that's what I'm trying to avoid until the end of the proc